package.json

```diff
@@ -70,7 +77,7 @@
     "prettier": "^1.19.1",
     "react-docgen-typescript-loader": "^3.4.0",
     "testcafe": "^1.7.1",
-    "typescript": "^3.7.4",
+    "typescript": "^3.9.7",
```

Comment:
Why not updating to the latest directly?

Reply:
ok